### PR TITLE
A safer way to position the clear icon

### PR DESCRIPTION
**Pull request by tomivirkki**
This change is [Reviewable](https://reviewable.io/reviews/vaadin/vaadin-date-picker/165)

### CODE REVIEW

File: test/basic.html [hunk 1]
```diff
@@ -232,6 +232,16 @@
         datepicker.open();
       });
 
+      it('should not change combobox width', function() {
```
Comment (jounik) [inline test/basic.html]: combobox mentioned :)

**Comment by tomivirkki** [comment]
*[test/basic.html, line 235 \[r1\]](https://reviewable.io:443/reviews/vaadin/vaadin-date-picker/165#-KFEj5jb2hTlmN_WXGu_-r1-235:-KFEl9YvwzeB1ceqNQs1:-499146009) ([raw file](https://github.com/vaadin/vaadin-date-picker/blob/8517a80f496e4e6969e81bad3c2c1d581c607373/test/basic.html#L235)):*
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-date-picker/165)*

**Comment by platosha** [comment]
:lgtm:

---

Reviewed 1 of 2 files at r1, 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-date-picker/165#-:-KFEyLIDBYP-MJ2dEwrQ:1417088721)*

**Comment by platosha** [comment]
:lgtm_cancel: 

---

Review status: all files reviewed at latest revision, 2 unresolved discussions.

---

*[vaadin-date-picker.html, line 153 \[r2\]](https://reviewable.io:443/reviews/vaadin/vaadin-date-picker/165#-KFJN0fKH8_IeYutlurG:-KFJN0fKH8_IeYutlurH:-256763588) ([raw file](https://github.com/vaadin/vaadin-date-picker/blob/693cee20f38f133d536774ddb1a0c89068a21ef6/vaadin-date-picker.html#L153)):*
Case mismatch in `#inputContainer` selector. See below, the input container has id in lower case: `id="inputcontainer"`.

Makes the style not be applied in Chrome, at least.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-date-picker/165#-:-KFJMx_JdnpPlXunMzgE:502544406)*

**Comment by tomivirkki** [comment]
*[vaadin-date-picker.html, line 153 \[r2\]](https://reviewable.io:443/reviews/vaadin/vaadin-date-picker/165#-KFJN0fKH8_IeYutlurG:-KFJSD3A5XoIZa57GSQd:264982969) ([raw file](https://github.com/vaadin/vaadin-date-picker/blob/693cee20f38f133d536774ddb1a0c89068a21ef6/vaadin-date-picker.html#L153)):*
Good catch! :D That's what you get from copy-pasteing stuff

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-date-picker/165)*

**Comment by platosha** [comment]
:lgtm:

---

Reviewed 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-date-picker/165#-:-KFJYc3CP5KjMaxBDmOY:1417088721)*

**Pull request merged**